src/crt_launch/SConscript

```diff
@@ -46,6 +47,10 @@ def scons():
 
     Import('env', 'prereqs', 'cart_lib', 'gurt_lib')
 
+    if not prereqs.check_component('ompi'):
+        print("No MPI installed...skipping crt_launch build");
+        return
```

Comment:
Do we want this to be non-fatal?

Reply:
This is the only way I was able to get builds on non-centos platforms to complete